### ISSUE TITLE
More completed than participants

### ISSUE THREAD

**Issue by dekz** [splitrb/split#294]
We ended up resetting out data by adding metadata (from my PR HAH!).

https://github.com/andrew/split/blob/master/lib/split/experiment.rb#L92-96

```ruby
        unless existing_alternatives == @alternatives.map(&:name) && existing_goals == @goals && existing_metadata == @metadata
          reset
          @alternatives.each(&:delete)
          delete_goals
          delete_metadata
          Split.redis.del(@name)
          @alternatives.reverse.each {|a| Split.redis.lpush(name, a.name)}
          @goals.reverse.each {|a| Split.redis.lpush(goals_key, a)} unless @goals.nil?
          Split.redis.set(metadata_key, @metadata.to_json) unless @metadata.nil?
```

I'm wondering what we should do about this. Since Split uses experiment name everywhere, even removes versions for the lookup. 

1. the version is incremented (accidentally)
2. the data has been reset (participant and completed)
3. there are still lagging people in the previous version experiment
4. Completed count > participant count
5. :boom: 

I believe any change to the experiment (alternative names, metadata changes) should result in a new version.  Should the lookup always contain the version number?

**Comment by divineforest** [comment]
@dekz the fix is in the master, pls try it

**Comment by andrew** [comment]
Fixed by #408
